src/index.ts

```diff
@@ -91,20 +91,25 @@ export {default as TouchableOpacity, TouchableOpacityProps} from './components/t
 export {default as View, ViewProps} from './components/view';
 export {default as Wizard, WizardProps, WizardStepProps, WizardStepStates, WizardStepConfig, WizardStepsConfig} from './components/wizard';
 
+/* ===== TestKit ===== */
+export {default as TextTestKit} from './components/text/Text.driver';
+export {default as ImageTestKit} from './components/image/Image.driver';
+export {default as ButtonTestKit} from './components/button/Button.driver';
```

Comment:
Because we haven't completed our typescript migration we're in a state where we maintain both `index.ts` and `index.js`. 
It's important to add the exports to the latter as well..